packages/dids/src/index.js

```diff
@@ -0,0 +1,34 @@
+import {v4 as uuidv4} from 'uuid';
+const didKeyDriver = require('@digitalbazaar/did-method-key').driver();
+
+const DID_DEFAULT_CONTEXT = [
+  'https://w3id.org/wallet/v1',
+  'https://w3id.org/did-resolution/v1',
+];
+export const DIDKeyManager = (function () {
```

Comment:
another option would be to use a class here, similar to the credentials management

Reply:
but I agree that just for one method would make more sense to just `export keypairToDidKeyDocument(...){ `

Reply:
I wrapped it because we might actually have more functions and I did not want to expose all of them.
It is just a pattern to make some stuffs private/public without actually using modifiers.

Reply:
pattern doesnt make sense because if its private you just dont export the method? you are here exporting basically like a static class whereby user has to do: `DIDManager.theMethod` instead of just `theMethod` which is cleaner, imo